macrobot/db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `macro` (
+  `channel_name` varchar(128) NOT NULL,
+  `macro_name` varchar(128) NOT NULL,
+  `macro_message` varchar(128) NOT NULL,
```

Comment:
maybe `macro_command`?

Reply:
I made this `message` in the help text: https://github.com/keybase/managed-bots/pull/225/files#diff-624b4a664eb4e50ba95dfd2836d02865R73

I thought message might be a better word, because although it _could_ be a command, it isn't necessarily one. I'm not sure though, open to suggestions.